src/retriers.js

```diff
@@ -0,0 +1,14 @@
+export let retriers = {
+  is5xx: (response) => {
```

Comment:
Probably! I think the intention here was to provide a few default retriers with some limited utility and also as documentation. Having said that, probably 503 / 504 would probably be the errors I'd actually want to retry as a consumer of this module.